src/Oddin.OddsFeedSdk/Oddin.OddsFeedSdk/API/PlayerCache.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.Caching;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using Oddin.OddsFeedSdk.API.Abstractions;
+using Oddin.OddsFeedSdk.API.Entities;
+using Oddin.OddsFeedSdk.API.Models;
+using Oddin.OddsFeedSdk.Common;
+
+namespace Oddin.OddsFeedSdk.API;
+
+internal class PlayerCache : IPlayerCache
+{
+    private static readonly ILogger _log = SdkLoggerFactory.GetLogger(typeof(PlayerCache));
+
+    private readonly IApiClient _apiClient;
+    private readonly MemoryCache _cache = new(nameof(PlayerCache));
+    private readonly TimeSpan _cacheTtl = TimeSpan.FromHours(12);
+    private readonly Semaphore _semaphore = new(1, 1);
+    private readonly IDisposable _subscription;
+
+    public PlayerCache(IApiClient apiClient)
+    {
+        _apiClient = apiClient;
+
+        _subscription = apiClient.SubscribeForClass<IRequestResult<object>>()
+            .Subscribe(response =>
+            {
+                if (response.Culture is null || response.Data is null)
+                    return;
+
+                var players = response.Data switch
+                {
+                    // No Player info in feed for now.
+                    _ => new player_profilePlayer[0]
+                };
```

Comment:
**[SOMETHING-MISSING]:** Is there something missing or it is planned to be filled later?